packages/slap/slap.3.0.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+maintainer: "abe@sf.ecei.tohoku.ac.jp"
+authors: [ "Akinori ABE <abe@sf.ecei.tohoku.ac.jp>" ]
+license: "LGPL-2.1 with OCaml linking exception"
+homepage: "http://akabe.github.io/slap/"
+dev-repo: "https://github.com/akabe/slap.git"
+bug-reports: "https://github.com/akabe/slap/issues"
+build: [
+  ["ocaml" "setup.ml" "-configure"
+   "--prefix" prefix
+   "--disable-ppx"  {ocaml-version < "4.02"}]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocamlfind" "remove" "slap"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "base-bigarray"
+  "base-bytes"
+  "ocamlfind" {>= "1.5.0"}
+  "lacaml" {>= "8.0.0"}
+  "cppo"
```

Comment:
`ocamlfind` and `cppo` can probably be marked as `{build}` dependencies as their upgrade is unlikely to change the behavior of this package and your package does not re-expose their types (unless you have some kind of build system extension or `cppo` run-time behavior...). I'm going to merge this regardless but it's something to keep in mind for the future.